radio/src/gui/480x272/model_mixes.cpp

```diff
@@ -440,6 +440,11 @@ bool menuModelMixAll(event_t event)
       }
       break;
     case EVT_KEY_BREAK(KEY_ENTER):
+      if (-1 == s_editMode)
```

Comment:
the more I see this -1, the more I think a constant would be nice (I know this is a bug of mine)